themes/default/content/blog/azure-container-solutions/index.md

```diff
@@ -0,0 +1,182 @@
+
+### What’s it good for?
+
+#### Extensibility
+
+Kubernetes is incredibly feature rich. Out of the box with AKS, you’ll get application self-healing capabilities via the mechanism of reconcile loops built into Kubernetes, the ability to run different types of workload via native Kubernetes resources such as StatefulSets, Deployments and Jobs, and a large amount of extensibility via Kubernetes’ Custom Resource Definitions capability.
+
+#### Familiarity
+
+Kubernetes also offers a familiar API regardless of the cloud provider you’re working with. If you’re new to AKS, but have used a managed Kubernetes service from another cloud provider, you might find it relatively easy to get your container up and running in AKS.
+
+### What’s not great about it?
+
+#### Complexity
+
+Kubernetes has a steep learning curve, and AKS is no exception. A lot of the knowledge needed to run and operate Kubernetes is handled by AKS itself, but you still have to have an in-depth understanding of how Kubernetes works to get your container running.
+
+In addition, the specification for a simple Kubernetes job can often be hundreds of lines of code. You have to learn not only how to run the container but also understand how Kubernetes exposes services to the world.
+
+### What we’d use it for
+
+The flexibility and extensibility offered by Kubernetes as a platform means it’s possible to run many types of workloads. Large companies have battle-tested Kubernetes (and by extension, AKS), running everything from web applications to machine learning.
+
+If you're able to invest time into learning and understanding the ecosystem and can dedicate people to maintaining and managing your Kubernetes implementation, AKS is a fantastic option for almost any workload you might consider.
+
+## Everything Else
+
+### Azure Red Hat OpenShift
+
+Azure Red Hat OpenShift is an enterprise grade implementation of Kubernetes developed by Red Hat. OpenShift builds on Kubernetes by adding support, enterprise-grade security, and Platform-as-a-Service integrations.
+
+If the idea of using Kubernetes for your container solutions seems appealing but you need service level agreements (SLAs) around support and integrations, Azure’s Red Hat OpenShift solution is a great option.
+
+### Azure Service Fabric
+
+Azure Service Fabric is a container service focused on distributed applications, like microservice architectures or complex distributed compute environments.
+
+If you’re planning on running a complex architecture, Azure Service Fabric can be a great fit. It provides more orchestration capabilities than Azure App Service. Service Fabric can also extend outside Azure's cloud into your on-premise data center if necessary.
+
+If your application has needs related to low latency, especially across distributed regions, Service Fabric is a great fit.
```

Comment:
Balance this with why it's not so great? I think at this point MS markets SF as "use it only if you really know what you are doing". It's definitely pretty hard to use AFAIK.